jbmc/src/java_bytecode/assignments_from_json.cpp

```diff
@@ -838,7 +838,9 @@ void assign_from_json(
   location.set_function(function_id);
   allocate_objectst allocate(ID_java, location, function_id, symbol_table);
   code_blockt body_rec;
-  const auto class_name = declaring_class(symbol_table.lookup_ref(function_id));
+  const symbolt *function_symbol = symbol_table.lookup(function_id);
```

Comment:
is lookup-ref vs. lookup-then-invariant really that different?

Reply:
yes, you get a reasonable message on failure.